pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1187,19 +1189,24 @@ func (t *Task) hasAllProgressReportingCompleted() (bool, error) {
 				t.Owner.Status.RunningPods = append(t.Owner.Status.RunningPods, podProgress)
 			case operation.Failed:
 				t.Owner.Status.FailedPods = append(t.Owner.Status.FailedPods, podProgress)
-			case dvmp.Status.PodPhase == corev1.PodSucceeded, operation.Succeeded:
+			case dvmp.Status.PodPhase == corev1.PodSucceeded:
 				t.Owner.Status.SuccessfulPods = append(t.Owner.Status.SuccessfulPods, podProgress)
-			case dvmp.Status.PodPhase == corev1.PodPending:
+			case dvmp.Status.PodPhase == corev1.PodPending, !operation.Failed:
 				t.Owner.Status.PendingPods = append(t.Owner.Status.PendingPods, podProgress)
-				pendingPods = append(pendingPods, fmt.Sprintf("%s/%s", podProgress.Namespace, podProgress.Name))
+				if dvmp.Status.CreationTimestamp != nil {
```

Comment:
Is dvmp creation timestamp is an accurate measure of the pod pending?

Reply:
@alaypatel07 that's Pod's creationTimestamp : https://github.com/konveyor/mig-controller/blob/f97cd1e001b122d6fbc73b95620b6a27f64bacb1/pkg/apis/migration/v1alpha1/directvolumemigrationprogress_types.go#L76